### PR TITLE
Jasper reports corrections

### PR DESCRIPTION
**Pull request by riadmen**
https://github.com/metasfresh/metasfresh/issues/16687

### CODE REVIEW

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Get_QtyPattern.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+DROP FUNCTION IF EXISTS report.Get_QtyPattern(numeric)
+;
+
+CREATE FUNCTION report.Get_QtyPattern(UOM_StdPrecision numeric) RETURNS text
+    LANGUAGE plpgsql
+AS
```
Comment (cristinamghita) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Get_QtyPattern.sql:28]: pls rename this to ```report.getQtyPattern```
Try to follow the example naming that we have.

**Review by cristinamghita — CHANGES_REQUESTED** [review]
minor comment about renaming

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Get_QtyPattern.sql (hunk 1 above)
Comment (riadmen) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Get_QtyPattern.sql]: done

**Review by cristinamghita — APPROVED** [review]
lgtm

**Pull request merged**